reusable/images_to_numpy.py

```diff
@@ -0,0 +1,90 @@
+import os
+import cv2
+import numpy as np
+from tqdm import tqdm
+from PIL import Image
+import matplotlib.pyplot as plt
+
+# Assuming Folder Structure
```

Comment:
Script should have only a single use-case of what it is going to do. 

See [this](https://github.com/hasibzunair/plugnplay/blob/main/reusable/letterbox_resize.py) for writing function description.

Reply:
Done!!